Python/Product/Analysis/Interpreter/Ast/AstPythonModule.cs

```diff
@@ -41,7 +41,9 @@ PythonLanguageVersion langVersion
             PythonLanguageVersion langVersion,
             string moduleFullName
         ) {
-            using (var stream = new FileStream(sourceFile, FileMode.Open, FileAccess.Read, FileShare.ReadWrite)) {
+            using(var stream = Directory.Exists(sourceFile) 
+                ? new MemoryStream()// Module without __init__.py, create empty stream
```

Comment:
So this is basically null-stream, right?

Reply:
Yes, like empty file signaling that this folder is a module.